src/test/java/com/ddd/moodof/acceptance/TagAcceptanceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.ddd.moodof.acceptance;
+
+import com.ddd.moodof.application.dto.TagDTO;
+import com.ddd.moodof.domain.model.user.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.util.List;
+import static com.ddd.moodof.adapter.presentation.TagController.API_TAG;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+public class TagAcceptanceTest extends AcceptanceTest{
+    private Long userId;
+
+    @Override
+    @BeforeEach
+    void setUp() {
+        super.setUp();
+        User user = signUp();
+        userId = user.getId();
+    }
+
+    @Test
+    public void 유저아이디_태그_전체_조회() throws Exception {
+        // given
+        태그_생성(userId, "tag1");
+        태그_생성(userId, "tag2");
+        태그_생성(userId, "tag3");
+        TagDTO.TagResponse tagResponse = 태그_생성(userId, "tag4");
+
+        // when
+        String uri =API_TAG;
+        List<TagDTO.TagResponse> response = getListWithLogin(uri, TagDTO.TagResponse.class, userId);
+
+        // then
+        assertAll(
+                () -> assertThat(response.size()).isEqualTo(4),
+                () -> assertThat(response.get(0).getName()).isEqualTo("tag1")
+        );
+    }
+
+    @Test
+    public void 태그를_생성한다() throws Exception {
+
+        // given
+
+        // when
+        TagDTO.TagResponse response = 태그_생성(userId, "name1");
+
+        // then
+        assertAll(
+                () -> assertThat(response.getId()).isNotNull(),
+                () -> assertThat(response.getUserId()).isEqualTo(userId),
+                () -> assertThat(response.getName()).isEqualTo("name1"),
+                () -> assertThat(response.getCreatedDate()).isNotNull(),
+                () -> assertThat(response.getLastModifiedDate()).isNotNull()
+        );
+    }
+    @Test
+    public void 태그를_삭제한다() throws Exception {
+        // given
+        태그_생성(userId, "name1");
+        TagDTO.TagResponse response = 태그_생성(userId, "name2");
+
+        // when
+        태그_리스트_출력(userId);
+        deleteWithLogin(API_TAG, response.getId(), userId);
+        태그_리스트_출력(userId);
+    }
+    @Test
+    public void 태그_수정() throws Exception {
+        // given
+        TagDTO.TagResponse createList = 태그_생성(userId, "name1");
+
+        // when
+        TagDTO.TagResponse updateList = 태그_수정(userId, createList.getId(),"name2");
```

Comment:
```suggestion
        TagDTO.TagResponse updateList = 태그_수정(createList.getId(), userId, "name2");
```

현재는 둘다 ID가 1이라서 통과한것 같습니다! 테스트 메소드의 동작을 봤을땐 인자가 서로 바뀌어야 할 것 같네요 🙂

Reply:
놓친부분이 있었네요 수정해서 반영해놓겠습니다 💁‍♂️